tasks/apt_install.yml

```diff
@@ -0,0 +1,71 @@
+- name: Add ruby repo
+  apt_repository:
+    repo: 'ppa:brightbox/ruby-ng'
```

Comment:
I really hate your mix of double and single quotes

Reply:
fixed